app/services/importers/ok2_explore/scrape_job.rb

```diff
@@ -0,0 +1,69 @@
+module Importers
+  module Ok2Explore
+    class ScrapeJob < ApplicationService
+      attr_reader :scrape_job, :bar
+
+      def initialize(scrape_job, bar: false)
+        @scrape_job = scrape_job
+        @bar = bar
+        super()
+      end
+
+      def perform
+        log "Scraping: #{scrape_job.id}"
+        begin
+          death_records = ::Ok2explore::Scraper.new(**scraper_args).perform
+        rescue ::Selenium::WebDriver::Error::TimeoutError
+          log 'Timeout occurred, skipping for now.'
+          return nil
```

Comment:
I hit the timeout once but when you're in the middle of a big run it's frustrating. If we want to switch this back to being in the worker we can do that too which might handle better.